crates/nu-std/lib/log.nu

```diff
@@ -1,49 +1,138 @@
-export def "log CRITICAL_LEVEL" [] { 50 }
-export def "log ERROR_LEVEL"    [] { 40 }
-export def "log WARNING_LEVEL"  [] { 30 }
-export def "log INFO_LEVEL"     [] { 20 }
-export def "log DEBUG_LEVEL"    [] { 10 }
-
-export def "log CRITICAL_LEVEL_PREFIX" [] { "CRT" }
-export def "log ERROR_LEVEL_PREFIX"    [] { "ERR" }
-export def "log WARNING_LEVEL_PREFIX"  [] { "WRN" }
-export def "log INFO_LEVEL_PREFIX"     [] { "INF" }
-export def "log DEBUG_LEVEL_PREFIX"    [] { "DBG" }   
-
-def parse-string-level [level: string] {
+export def "log CRITICAL_LEVEL" [] {
+	50
+}
+
+export def "log ERROR_LEVEL" [] {
+	40
+}
+
+export def "log WARNING_LEVEL" [] {
+	30
+}
+
+export def "log INFO_LEVEL" [] {
+	20
+}
+
+export def "log DEBUG_LEVEL" [] {
+	10
+}
+
+def parse-string-level [
+    level: string
+] {
     (
-        if ($level in [(log CRITICAL_LEVEL_PREFIX) "CRIT" "CRITICAL"]) {
+        if $level == "CRITICAL" {
             log CRITICAL_LEVEL
-        } else if ($level in [(log ERROR_LEVEL_PREFIX) "ERROR" ]) {
+        } else if $level == "CRIT" {
+            log CRITICAL_LEVEL
+        } else if $level == "ERROR" {
             log ERROR_LEVEL
-        } else if ($level in [(log WARNING_LEVEL_PREFIX) "WARN" "WARNING"]) {
+        } else if $level == "WARNING" {
+            log WARNING_LEVEL
+        } else if $level == "WARN" {
             log WARNING_LEVEL
-        } else if ($level in [(log DEBUG_LEVEL_PREFIX) "DEBUG"]) {
+        } else if $level == "INFO" {
+            log INFO_LEVEL
+        } else if $level == "DEBUG" {
             log DEBUG_LEVEL
         } else {
             log INFO_LEVEL
         }
     )
 }
 
-def parse-int-level [level: int] {
+export def "log CRITICAL_LEVEL_PREFIX" [
+    --short (-s)
```

Comment:
I would also add tests for the short prefixes :)

Reply:
@EmilySeville7cfg 
one such test might just be
```bash
# in test_logger.nu
use std "assert equal"
use std

export def test_short_prefixes [] {
    assert equal (std log CRITICAL_LEVEL_PREFIX) "CRT"
    assert equal (std log CRITICAL_LEVEL_PREFIX --short) "C"
    ...
}
```

Reply:
I was rather thinking about testing if `log debug --short` will indeed result in `D` instead of `DBG` in the log string. But yeah, explicit tests for the `std log *_PREFIX` commands are useful too.

Reply:
For some reason, I see `module not found` error when I open tests for loggers. 🤔 

![image](https://user-images.githubusercontent.com/42812113/236638951-566f5b9d-b671-428c-86f4-50ffa1fe650c.png)

Reply:
@EmilySeville7cfg to run the tests:

```nu
use toolkit.nu
NU_LOG_LEVEL=DEBUG toolkit test stdlib
```

Reply:
Done. :)

Reply:
> I was rather thinking about testing if `log debug --short` will indeed result in `D` instead of `DBG` in the log string. But yeah, explicit tests for the `std log *_PREFIX` commands are useful too.

How about that?

Reply:
Just wrote `*_short` tests. Is it what you expected?

Reply:
I like it, thanks :)